### PR TITLE
Replace internal pm with workspace controls

### PR DESCRIPTION
**Pull request by seanbrookes**
connected to: #1484

### CODE REVIEW

**Comment by Setogit** [comment]
connect to https://github.com/strongloop/strong-arc/issues/582

File: client/test/e2e/arc/views/app-controller-view.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        EC.presenceOf(this.statusRunningDisplay),
+        10000);
+      browser.driver.wait(
+        EC.presenceOf(this.appStopButton),
+        10000);
+      this.appStopButton.click();
+      browser.driver.wait(
+        EC.presenceOf(this.statusStoppedDisplay),
+        10000);
+    };
```
Comment (anthonyettinger) [inline client/test/e2e/arc/views/app-controller-view.js]: I've been adding a `var wait = 1000;` and use that so its easy to change them all in a particular spec.

File: client/test/e2e/arc/views/header-view.js [hunk 2]
```diff
@@ -18,6 +20,17 @@ var ArcHeaderView = (function () {
       browser.driver.sleep(500);
       browser.waitForAngular();
     };
+
+    this.openAppController = function() {
+      browser.driver.wait(
+        EC.presenceOf(this.appControllerDropdown),
+        10000);
+      browser.executeScript('window.scroll(0, 0);');
+      browser.sleep(100);
```
Comment (anthonyettinger) [inline client/test/e2e/arc/views/header-view.js]: use `browser.driver.sleep`

**Comment by anthonyettinger** [comment]
Few comments, otherwise LGTM

**Comment by seanbrookes** [comment]
test please

File: client/test/e2e/arc/app-controller.spec.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+var ArcViews = require('../arc/views/');
+
+var EC = protractor.ExpectedConditions;
+
+describe('app-controller-test', function() {
+  fdescribe('app-start', function() {
```
Comment (rmg) [inline client/test/e2e/arc/app-controller.spec.js]: What is `fdescribe`? If it isn't a typo, then it needs to be added to `client/test/.jshintrc` as a registered global.

Comment (seanbrookes) [inline client/test/e2e/arc/app-controller.spec.js]: accidentally left an 'isolate' test notation

Comment (anthonyettinger) [inline client/test/e2e/arc/app-controller.spec.js]: fdescribe just runs that one set of tests. looks like he forgot to remove it.

**Comment by seanbrookes** [comment]
test please

**Comment by seanbrookes** [comment]
test please

**Pull request merged**